apps/www/src/registry/default/plate-ui/inline-date-element.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+import React from 'react';
+
+import type { TInlineDateElement } from '@udecode/plate-date';
+
+import { cn } from '@udecode/cn';
+import {
+  PlateElement,
+  findNodePath,
+  setNodes,
+  useEditorRef,
+  useElement,
+  withRef,
+} from '@udecode/plate-common';
+import { format } from 'date-fns';
+
+import { Calendar } from './calendar';
+import { Popover, PopoverContent, PopoverTrigger } from './popover';
+
+export const InlineDateElement = withRef<typeof PlateElement>(
+  ({ children, className, ...props }, ref) => {
+    const element = useElement<TInlineDateElement>();
+
+    const editor = useEditorRef();
+    const path = findNodePath(editor, element);
```

Comment:
I recommend calling `findNodePath` inside the callback where the path is used. This should be more efficient and reduce the risk of a stale path being used.

Reply:
Oh, sorry, I mean inside the `onSelect` callback

Reply:
Oh, I understand now. I misunderstood before. My mistake.Thanks！